app/src/main/res/layout/activity_adopt_activity.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".adopt_activity">
+
+    <android.support.v4.view.ViewPager
+        android:id="@+id/dogimage"
+        android:layout_width="match_parent"
+        android:layout_height="276dp" />
```

Comment:
Almost never should a view have a width or height with dp. If you put it on a tablet, it will look ridiculous. You want it to be scalable.